PowerPointLabs/PowerPointLabs/ActionFramework/TooltipsLab/Create/CreateMenu/CreateMenuEnabledHandler.cs

```diff
@@ -0,0 +1,19 @@
+﻿
+using PowerPointLabs.ActionFramework.Common.Attribute;
+using PowerPointLabs.ActionFramework.Common.Extension;
+using PowerPointLabs.ActionFramework.Common.Interface;
+using PowerPointLabs.Models;
+using PowerPointLabs.TextCollection;
+
+namespace PowerPointLabs.ActionFramework.TooltipsLab
+{
+    [ExportEnabledRibbonId(TooltipsLabText.CreateMenuTag)]
+    class CreateMenuEnabledHandler : EnabledHandler
+    {
+        protected override bool GetEnabled(string ribbonId)
+        {
+            PowerPointSlide currentSlide = this.GetCurrentSlide();
+            return currentSlide != null;
```

Comment:
Hi, just noticed a small issue here. A `null reference exception` will be thrown when the code is running from Visual Studio if current slide is null. The exception is caused by the underlying implementation of `this.GetCurrentSlide()` function.  

Suggested way to fix this:
- Use a boolean variable to indicate whether `createMenu` should be enabled, and alter the variable value when slide selection is changed. In this way, empty slide selection can be handled explicitly. OR
- Change `PowerPointCurrentPresentationInfo.CurrentSlide` to handle null value explicitly.

Reply:
@LiXiaoooowei How do you replicate the problem? I can't seem to get the exception.

Reply:
> @LiXiaoooowei How do you replicate the problem? I can't seem to get the exception.

I think it probably depends on your exception settings for visual studio. I set it to break when `C++ Exceptions Platform::NullReferenceException` is thrown.